components/web_discovery/browser/hash_detection.cc

```diff
@@ -0,0 +1,1489 @@
+             -4.414714369803455,  -3.0879080140350674, -5.469976217597951,
+             -3.790334046490602,  -5.366435538657111,  -3.235086137375257,
+             -5.727805326900051,  -5.408282648592611,  -4.971959552124542,
+             -5.070590155566169,  -5.076706382583605,  -6.181472536826093,
+             -5.921961341341008,  -4.9999725883522155, -5.5070174892783,
+             -1.5568628976878147, -5.5070174892783,    -6.110013572843948,
+             -4.0678979887767985, -5.089052218405905,  -5.200643283814367,
+             -3.6661684098537215, -4.557328477002297,  -5.479108701161223,
+             -5.159821289294111,  -5.479108701161223,  -5.616579691789827,
+             -3.7152580200502454, -4.23556238777078,   -4.212031890360586,
+             -3.866007780968386,  -4.0678979887767985, -4.4633463186414915,
+             -4.142178063159669,  -4.318254103615893,  -4.276053749125516,
+             -4.772293881872614,  -4.332726136224427,  -4.11133338780857,
+             -3.9089648990693253, -4.309670359924501,  -3.9127889955077286,
+             -4.338574106106851,  -4.528549512452254,  -4.149433234040841,
+             -3.5292744136497762, -3.6009751629760713, -4.350373653038005,
+             -4.828122331425556,  -4.047963773875981,  -4.665125047458004,
+             -4.673288358097166,  -4.633122316371831,
+         },
+         {
+             -4.806719860900339,  -3.6507473420444714, -4.429814962058902,
+             -4.347330952102998,  -4.3949170222076255, -4.698366552912107,
+             -4.809448375553543,  -4.35423944244681,   -4.534137594904308,
+             -4.3611959922401695, -2.871768725634745,  -4.598399529122678,
+             -5.564587232141456,  -5.519124858064699,  -2.4250566837898457,
+             -5.688201188108633,  -5.967149580591659,  -5.01834957015221,
+             -2.505498094634157,  -5.3939617151106924, -6.3813483686685775,
+             -5.360223575478843,  -5.821732580733155,  -5.742268409378909,
+             -2.5427453130265225, -6.0299504818306895, -5.949907774157153,
+             -6.048642614842842,  -6.106911522966818,  -6.522426966928483,
+             -1.9368412502165815, -5.384205540165328,  -3.2469989725553616,
+             -5.166904264475346,  -4.662348253722951,  -5.93295821584338,
+             -4.052787789271272,  -3.8964417188805776, -4.725629703670472,
+             -3.994277013931218,  -3.570836586604418,  -4.883136091344702,
+             -4.20182336843176,   -4.894970548991704,  -4.1342203512338624,
+             -5.159122124033291,  -4.442985425248647,  -4.740819869164446,
+             -4.583031498894364,  -4.589588899440523,  -4.4058064220068935,
+             -4.87144005158151,   -4.8541485544714496, -4.806719860900339,
+             -4.241282205172307,  -4.937530163410501,  -4.382360803432213,
+             -5.0116381355642226, -4.368201274828579,  -4.782492565565015,
+             -4.544565218066568,  -4.4563316672015425,
+         }};
```

Comment:
let's put the numbers into a separate hash_detection-inc.cc that you can `#include` into this file